feature/lldp/otg_tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -41,8 +47,6 @@ type lldpTestParameters struct {
 
 type lldpNeighbors struct {
 	systemName    string
-	portId        string
```

Comment:
Why are we removing portID/type here? According to the README these are values we want to check.

Reply:
Added portId and portIdType verifcation for the LLDP Neighbor telemetry to satisfy these oc path requirements from README
```
/lldp/interfaces/interface/neighbors/neighbor/state/port-id
/lldp/interfaces/interface/neighbors/neighbor/state/port-id-subtype

Reply:
See two test logs below 
[dut 1 log](https://drive.google.com/file/d/1lB0D7nC9GbAZJjDwnISsi3D2zUpqRY92/view?usp=drive_link)
[dut 2 log](https://drive.google.com/file/d/1jqZwqM1JiLuKMf6dnBveT8Dyuhi7MbLk/view?usp=drive_link)

Reply:
Is there a reason that we want these as two separate types (vs. just having the different forms of the checks on the DUT/ATE side)?

We are also using the naming `expOtgLLDPNeighbor := lldpNeighbors` below which I think is a little confusing since with the type `otgLLDPNeighbors`.

Reply:
Simplified to a single lldpNeighbors struct.